src/pages/Profile/components/UserPosts/UserPostFilter/UserLikePostButton.tsx

```diff
@@ -0,0 +1,22 @@
+import * as S from "./UserPostFilter.Styles"
+
+interface UserLikePostButtonProps {
+  onClick: () => void
+  $isSelected: boolean
+}
+
+const UserLikePostButton = ({
+  onClick,
+  $isSelected,
+}: UserLikePostButtonProps) => {
+  return (
+    <S.UserLikePostsButton
+      $isSelected={$isSelected}
+      onClick={onClick}
+    >
+      좋아요한 게시글
+    </S.UserLikePostsButton>
+  )
+}
+
+export default UserLikePostButton
```

Comment:
현재는 UserCreatePostButton과 UserLikePostButton 에 공통적인 요소만 들어가서 2개의 컴포넌트를 1개로 합치는 것도 좋을 것 같습니다! 🤔

바꾸신다면 이렇게 바꿔보시면 될 것 같습니다! 👍 
그리고 밑에 UserPostFilter 컴포넌트에서 UserCreatePostButton과 UserLikePostButton 대신 바꾼 컴포넌트를 쓰면 될 것 같습니다!
```suggestion
interface UserFilteredPostButtonProps {
  onClick: () => void
  $isSelected: boolean
  title: string
}

const UserFilteredPostButton = ({
  onClick,
  $isSelected,
  title
}: UserFilteredPostButtonProps) => {
  return (
    <S.UserFilteredPostButton
      $isSelected={$isSelected}
      onClick={onClick}
    >
      {title}
    </S.UserLikePostsButton>
  )
}

export default UserFilteredPostButton
```